commands.js

```diff
@@ -1892,6 +1892,28 @@ exports.commands = {
 	},
 	hidetexthelp: ["/hidetext [username] - Removes a locked or banned user's messages from chat (includes users banned from the room). Requires: % (global only), @ * # & ~"],
 
+	unlogtext: function (target, room, user) {
+		if (!target) return this.parse('/help unlogtext');
+		this.splitTarget(target);
+		let targetUser = this.targetUser;
+		let name = this.targetUserName;
+		if (!user.can("rangeban")) {
```

Comment:
I based the permissions checking and other setup checks on the hidetext command, which uses user.can("roomban"). If I need to make a new permission I can but I think the two actions are related closely enough that sharing the permission should suffice.